cloud_testing/kotlin_poc/src/test/kotlin/ftl/android/AndroidCatalogTest.kt

```diff
@@ -21,4 +21,13 @@ class AndroidCatalogTest {
         assertEquals(IncompatibleModelVersion, AndroidCatalog.supportedDeviceConfig("NexusLowRes", "21"))
         assertEquals(SupportedDeviceConfig, AndroidCatalog.supportedDeviceConfig("NexusLowRes", "23"))
     }
+
+    @Test
+    fun validateIsVirtualDevice() {
+        if (bitrise) return
```

Comment:
with mocking, are we able to use a fake AndroidCatalog on bitrise? I worry about too many `if (bitrise) return` means we're not running the tests in CI at all.

Reply:
the mock server handles the android catalog now, but I got some other credentials error when I tried to allow these tests to run.

Reply:
https://app.bitrise.io/build/9ab1863db270de36 was the build that errored.
`The Application Default Credentials are not available.`

Reply:
```
 init {
        FtlConstants.useMock = true
    }
```

may fix that and activate the mock credential logic